biogeophys/FatesHydroWTFMod.F90

```diff
@@ -1078,7 +1709,7 @@ function dftcdpsi_from_psi_tfs(this,psi) result(dftcdpsi)
        dftcdpsi = 0._r8
     else
        ftc = 1._r8/(1._r8 + (psi/this%p50)**this%avuln)
-       if(ftc<min_ftc) then
+       if(ftc<min_ftc*10) then
```

Comment:
Please add a comment describing the change here with author name and date

Reply:
This was for a test, it doesn't need to be updated.

Reply:
ok